pulsar-client/src/main/java/org/apache/pulsar/client/impl/HttpLookupService.java

```diff
@@ -176,5 +176,12 @@ public void close() throws Exception {
         httpClient.close();
     }
 
+    private long bytesToLong(byte[] bytes) {
+        schemaVersionConvertBuffer.clear();
+        schemaVersionConvertBuffer.put(bytes, 0, bytes.length);
```

Comment:
hmm. this would result in copying memory. and it is not thread-safe.

I would suggest just using `ByteBuffer.wrap(bytes)` to get a ByteBuffer and then read the long number.